study.md

```diff
@@ -54,5 +54,5 @@ like enumerables?  How are they different?  Please compare and contrast these
 types.
 
 ```md
-<!-- your answer here -->
+Both are looking at data in a order. enumerables have different methods and streams can have unlimit data. Please explain or go over this in class. I have never been so confused on a study repo.
```

Comment:
Streams should generally be treated as infinite, even though they can be finite, and enumerables should generally be treated as finite, even though they can be infinite.